doc/nif-spec.md

```diff
@@ -194,13 +194,15 @@ Grammar:
 
 ```
 Digit ::= [0-9]
-NumberSuffix ::= [a-z]+ [0-9a-z]*  # suffixes can only contain lowercase letters
+NumberSuffix ::= '"' [a-z]+ [0-9a-z]* '"' # suffixes can only contain lowercase letters
```

Comment:
NumberSuffix can only be `u` for "unsigned", nothing else exists, the `suf` tree is not part of the lexing step.

Reply:
>  the suf tree is not part of the lexing step.

What does this mean? Do you mean the suf tree is handled by nifc/gear2/...?

Reply:
Yes.

Reply:
eh, I cannot compile gear2

Reply:
I have fixed gear2/gen2nif. `genToNim` seems to be incomplete